### DISCUSSION TITLE
VScode Indentação

### DISCUSSION THREAD

**Discussion by Al3xFlores** [frontendbr/forum#1688] [Perguntas e Respostas]
Ola devs!
Estou enfrentando o seguinte problema com VS.code.
Eu havia feito uma configuração para indentar o código automaticamente depois de salvar,
porem isso está interferido no meu estudo e queria desfazer só que não me lembro 
como fiz para desfazer kkkkkk
alguém sabe ?

**Comment by wilsonneto-swe** [comment]
Aperta Ctrl + "," para abrir as configurações, acha as propriedades:
        "editor.tabSize": 2,
        "editor.formatOnSave": true

Se quiser mudar o "tamanho do tab" é a primeira, se quiser que ele pare de formatar ao salvar é a segunda...

Abraços!

**Comment by Al3xFlores** [comment]
> editor.formatOnSave
Vlw man consegui conforme suas orientações, muito obrigado

**Comment by tarciozemel** [comment]
Bastante projetos usam [EditorConfig](https://editorconfig.org/). De repente, vale a pena conhecer.
